appgate/types.py

```diff
@@ -276,6 +349,20 @@ class Context:
     device_id: Optional[str] = attrib(default=None)
 
 
+@attrs()
```

Comment:
Not really related to this PR but maybe its time to start using the new style attrs class definitions, its closer to dataclasses so no need to use `attrib()` if not necessary.
See https://www.attrs.org/en/stable/examples.html
```
@define
class Coordinates:
    x: int
    y: int
```

The attrs/attrib stuff is the legacy way now if I understand this correctly.